packages/browser/src/login/oidc/incomingRedirectHandler/AuthCodeRedirectHandler.ts

```diff
@@ -105,6 +105,14 @@ export class AuthCodeRedirectHandler implements IIncomingRedirectHandler {
       this.issuerConfigFetcher
     );
 
+    const iss = url.searchParams.get("iss");
+
+    if (typeof iss === "string" && iss !== issuerConfig.issuer) {
+      throw new Error(
+        `The value of the iss parameter (${iss}) does not match the issuer identifier of the authorization server (${issuerConfig.issuer}). See [rfc9207](https://www.rfc-editor.org/rfc/rfc9207.html#section-2.3-3.1.1)`
```

Comment:
Now that. That is a well written damn error message.

![](https://media3.giphy.com/media/DDIIF8jCjh1srju4zN/200.gif?cid=5a38a5a2ih6bqur82nlouemszuaut4s4ru9cqv4v4a4f2bef&rid=200.gif&ct=g)